web/views.py

```
@@ -501,6 +501,17 @@ def profile(request):
 
     return render(request, "profile.html", context)
 
+@login_required
+def notification_preferences(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `notification_preferences`

(ANN201)